### PR TITLE
feat-Wishlist

### PR DESCRIPTION
**Pull request by iamsaurav28**
Screenshot (99)

### CODE REVIEW

File: comp-layout/wishlist.html [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+     </div>
+</header>
+
+
+     
+      <!--product feature-->
+
+
+    
+          <div class="row wishlist">
```
Comment (shuntzhere) [inline comp-layout/wishlist.html]: Check these classes.

File: comp-layout/wishlist.html [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+<link rel="stylesheet" href="/stylesheet/main.css">
+
+     <title>Wishlist-page</title>
+</head>
+<body>
+
+
+     <header class="home-header">
+     <div class="home-container">
+          <div class="navbar">
```
Comment (shuntzhere) [inline comp-layout/wishlist.html]: Check for the purpose of this div.

File: comp-layout/wishlist.html [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+                   <h4>CLASSIC WATCH</h4>
+                   <div class="rating">
+                        <i class="fa fa-star"></i>
+                        <i class="fa fa-star"></i>
+                        <i class="fa fa-star"></i>
+                        <i class="fa fa-star"></i>
+                        <i class="fa fa-star-o"></i>
+                        <i class="fa fa-heart heart"></i>
+                   </div>
+                   <P  class="price">$150.00</P>
```
Comment (shuntzhere) [inline comp-layout/wishlist.html]: Check this tag

File: comp-layout/wishlist.html [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+                  <h4>BLUE PLAIN T-SHIRT</h4>
+                  <div class="rating">
+                       <i class="fa fa-star"></i>
+                       <i class="fa fa-star"></i>
+                       <i class="fa fa-star"></i>
+                       <i class="fa fa-star"></i>
+                       <i class="fa fa-star-o"></i>
+                       <i class="fa fa-heart heart"></i>
+                  </div>
+                  <P  class="price">$150.00</P>
```
Comment (shuntzhere) [inline comp-layout/wishlist.html]: Check this p tag

File: comp-layout/wishlist.html [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+
+</div>
+      
+ <!--- footer-->
+
+
+ <div class="foot">
+     <footer class="footer">
+          <div class="footer-container">
+               <div class="webmaker">MADE BY SAURAV DARJI</div>
```
Comment (shuntzhere) [inline comp-layout/wishlist.html:106]: Check if this div is serving any purpose.

**Review by shuntzhere — APPROVED** [review]
Overall code looks good, suggested a few changes do give a look.

**Review by Sameony — APPROVED** [review]
nice and clean code, you can merge

File: comp-layout/wishlist.html (hunk 1 above)
Comment (iamsaurav28) [reply comp-layout/wishlist.html]: yes done

File: comp-layout/wishlist.html (hunk 2 above)
Comment (iamsaurav28) [reply comp-layout/wishlist.html]: okay done

File: comp-layout/wishlist.html (hunk 4 above)
Comment (iamsaurav28) [reply comp-layout/wishlist.html]: done

File: comp-layout/wishlist.html (hunk 5 above)
Comment (iamsaurav28) [reply comp-layout/wishlist.html:106]: done

**Pull request merged**